capella2polarion/__main__.py

```diff
@@ -239,5 +238,122 @@ def render_documents(
         polarion_worker.update_documents(project_data.updated_docs, project)
 
 
+@cli.command()
+@click.option(
+    "--plugin-config-file",
+    type=click.File(mode="r", encoding="utf8"),
+    required=True,
+    envvar="CAPELLA2POLARION_SYNCHRONIZE_CONFIG",
+)
+@click.option(
+    "--document-rendering-config",
+    type=click.File(mode="r", encoding="utf8"),
+    envvar="CAPELLA2POLARION_DOCUMENT_CONFIG",
+)
+@click.option(
+    "--overwrite-layouts",
+    is_flag=True,
+    default=False,
+    envvar="CAPELLA2POLARION_OVERWRITE_LAYOUTS",
+)
+@click.option(
+    "--overwrite-numbering",
+    is_flag=True,
+    default=False,
+    envvar="CAPELLA2POLARION_OVERWRITE_NUMBERING",
+)
+@click.option(
+    "--synchronize-config",
+    type=click.File(mode="r", encoding="utf8"),
+    envvar="CAPELLA2POLARION_SYNCHRONIZE_CONFIG",
+)
+@click.option(
+    "--force-update",
+    is_flag=True,
+    envvar="CAPELLA2POLARION_FORCE_UPDATE",
+    default=False,
+)
+@click.option(
+    "--type-prefix",
+    type=str,
+    envvar="CAPELLA2POLARION_TYPE_PREFIX",
+    default="",
+)
+@click.option(
+    "--role-prefix",
+    type=str,
+    envvar="CAPELLA2POLARION_ROLE_PREFIX",
+    default="",
+)
+@click.option(
+    "--grouped-links-custom-fields / --no-grouped-links-custom-fields",
+    envvar="CAPELLA2POLARION_GROUPED_LINKS_CUSTOM_FIELDS",
+    is_flag=True,
+    default=True,
+)
+@click.option(
+    "--generate-figure-captions",
+    envvar="CAPELLA2POLARION_GENERATE_FIGURE_CAPTIONS",
+    is_flag=True,
+    default=False,
+)
+@click.pass_context
+def run_plugins(
+    ctx: click.core.Context,
+    plugin_config_file: typing.TextIO,
+    document_rendering_config: typing.TextIO | None,
+    overwrite_layouts: bool,
+    overwrite_numbering: bool,
+    synchronize_config: typing.TextIO | None,
+    force_update: bool,
+    type_prefix: str,
+    role_prefix: str,
+    grouped_links_custom_fields: bool,
+    generate_figure_captions: bool,
+) -> None:
```

Comment:
I don't really like that this command has so many parameters.

Reply:
It is possible to use kwargs, but mypy does not like this. We could suppress that error, but I think there is a risk that we miss issues due to that in the future. So I would recommend to keep it as is